bindings/py/tests/SDR_Metrics_test.py

```diff
@@ -0,0 +1,123 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2019, David McDougall
+# The following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""Unit tests for SDR_Metrics python bindings"""
+
+import pickle
+import numpy as np
+import unittest
+import pytest
+
+from nupic.bindings.algorithms import SDR, SDR_Proxy
+from nupic.bindings.algorithms import SDR_Sparsity, SDR_ActivationFrequency, SDR_Overlap, SDR_Metrics
+
+class SdrMetricsTest(unittest.TestCase):
+    def testSparsityExample(self):
+        A = SDR( dimensions = 1000 )
```

Comment:
nit, but imho you might consider also removing the SDR(int) binding. Keep it only vector/list, I was still wondering here if it's a 1000 dimensional SDR, or 1D with width 1000. SDR(dimensions = [1000]) is imho much clearer.